packages/e2e/cypress/e2e/app/sqlRunner.cy.ts

```diff
@@ -125,7 +125,7 @@ describe('SQL Runner (new)', () => {
         cy.get('div[role="option"]').contains('first_name').click();
         cy.get('.echarts-for-react')
             .find('text')
-            .contains('Customer id sum aaron')
+            .contains('Customer id sum frances')
```

Comment:
now that we get more columns, the legend doesn't include aaron on the first page 
so I updated this assertion.

Compared the same chart with this PR's changes and without

with
<img width="563" alt="Screenshot 2025-04-16 at 12 00 40" src="https://github.com/user-attachments/assets/ea8f2c6a-81b2-49cd-8a2a-cab1b23a20da" />
You can test here with querying customers and doing the config as on the screenshot

https://lightdash-pr-14434.onrender.com/projects/3675b69e-8324-4110-bdca-059031aa8da3/sql-runner

without
<img width="1513" alt="Screenshot 2025-04-16 at 12 00 35" src="https://github.com/user-attachments/assets/80794e65-0aa3-41f6-84d8-ea67b144d12b" />

Reply:
wdyt @magnew ? It's definitely a change we need to make but could cause confusion to users. Does it make sense to have a headway announcement?